backend/editor/entries.py

```diff
@@ -21,6 +21,91 @@ def shutdown_db():
     session.close()
     driver.close()
 
+def get_label(id):
+    """
+    Helper function for getting the label for a given id
+    """
+    if (id.startswith('stopword')): return 'STOPWORDS'
+    elif (id.startswith('synonym')): return 'SYNONYMS'
+    elif (id.startswith('__header__') or id.startswith('__footer__')): return 'TEXT'
+    else: return 'ENTRY'
+
+def create_node(label, entry, mainLanguageCode):
+    """
+    Helper function used for creating a node with given id and label
+    """
+    query = [f"""CREATE (n:{label})\n"""]
+
+    # Build all basic keys of a node
+    if (label == "ENTRY"):
+        canonicalTag = entry[3:]
+        query.append(f""" SET n.main_language = "{mainLanguageCode}" """) # Required for only an entry
+    else:
+        canonicalTag = ""
+
+    query.append(f""" SET n.id = $id """)
+    query.append(f""" SET n.tags_{mainLanguageCode} = ["{canonicalTag}"] """)
+    query.append(f""" SET n.preceding_lines = [] """)
+
+    result = session.run(" ".join(query), {"id": entry})
+    return result
+
+def add_node_to_end(label, entry):
+    """
+    Helper function which adds an existing node to end of taxonomy
+    """
+    # Delete relationship between current last node and __footer__
+    query = [f"""MATCH (a)-[r:is_before]->(b:TEXT) WHERE b.id = "__footer__" DELETE r\n"""]

```

Comment:
Here and below, I think we would gain in clarity by avoiding `a`,  `b` and `n`. (because it's a bit hard to read otherwise)

We could instead use `last_node`, `footer` and `new_node`.